docs/src/installation.md

```diff
@@ -0,0 +1,153 @@
+
+## Quickstart with Scala CLI
+
+The easiest way to install Chisel is to [install Scala CLI](https://scala-cli.virtuslab.org/install) to build and run the Chisel template:
+
+```bash
+wget https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala
+scala-cli chisel-template.scala
+```
+
+The Chisel template is a simple, single-file example of Chisel that emits Verilog to the screen.
+
+:::tip
+
+While more complex projects often use a build tool like SBT or Mill as described below,
+we still highly recommend using Scala CLI with the template for experimentation and
+writing small snippets to share with others.
+
+:::
+
+## Dependencies
+
+As described above, Scala CLI is a great "batteries included" way to use Chisel.
+It will automatically download and manage all dependencies of Chisel _including the JVM_.
+More complex projects will require the user to install the JDK and a build tool.
+
+:::info
+
+Note that each of these dependencies are projects with their own installation instructions.
+Please treat the commands below as suggestions and not directives.
+
+:::
+
+### Java Development Kit (JDK)
+
+Scala runs on the Java Virtual Machine (JVM), so it is necessary to install a JDK to use Chisel.
+Chisel works on any version of Java newer than Java; however, we recommend using an LTS release version 17 or newer.
+Note that Scala CLI requires Java 17 or newer so unless your system installation of Java is at least version 17, Scala CLI will download Java 17 for its own use.
+
+You can install any distribution of the JDK you prefer.
+Eclipse Adoptium Temurin is a good option with support for all platforms: https://adoptium.net/installation
+
+#### Ubuntu
+
+```sh
+apt install -y openjdk-17-jdk
+```
+
+#### MacOS
+
+Using [MacPorts](https://www.macports.org):
+```sh
+sudo port install openjdk17-temurin
+```
+
+Using [Homebrew](https://brew.sh):
+```sh
+brew tap homebrew/cask-versions
+brew install --cask temurin17
+```
+
+#### Windows
+
+Using [Scoop](https://scoop.sh):
+```sh
+scoop install temurin17-jdk
+```
+
+Windows users may also prefer using an [installer](https://adoptium.net/temurin/releases/?version=17&os=windows&arch=x86&package=jdk).
+
+
+### Build Tools
+
+Scala CLI is only intended for projects made up of a single to a handful of files.
+Larger projects need a build tool.
+
+#### Mill
+
+[Mill](https://mill-build.com) is a modern Scala build tool with simple syntax and a better command-line experience than SBT.
+We recommend Chisel users use Mill.
+
+For detailed instructions, please see the [Mill documentation](https://mill-build.com/mill/Intro_to_Mill.html).
+
+##### Linux and MacOS
+
+The easiest way to use Mill is with the Mill Wrapper Script `millw`:
+
+```sh
+curl -L https://raw.githubusercontent.com/lefou/millw/0.4.11/millw > mill && chmod +x mill
+```
+You can then move this script to a global install location
+```sh
+sudo mv mill /usr/local/bin/
+```
```

Comment:
Advising uses to install something by moving things into a root-owned location is questionable. However, this is also what scala-cli suggests.

Reply:
AFAICT it's pretty universal that Linux-y things suggest you install things in `/usr/local/bin` and if there's a reason you shouldn't it seems that the reader is expected to know this and adjust accordingly. I don't have a strong opinion, I'm just following typical style.